app/components/sections/hero-section.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react'
+import {ArrowButton} from '../arrow-button'
+import {Grid} from '../grid'
+import {H1} from '../title'
+import {Button} from '../button'
+
+// Note that the image overlaps the right edge of the grid by `8vw`. This `8vw`
+// needs to stay in sync with the `10vw` margins of the grid component.
+function HeroSection() {
+  return (
+    <Grid className="h-full">
+      <div className="relative col-span-full px-4 lg:col-span-6 lg:col-start-7 lg:px-0 lg:h-full">
+        <div className="lg:right-[-8vw] bottom-0 left-0 right-0 top-0 flex items-center justify-center lg:absolute">
+          <img
+            alt=""
+            className="max-w-[80vh] w-full h-auto object-contain"
```

Comment:
This `max-w-[8-vh]` should probably be dropped. Need to spend a bit more time on this to get it right.